src/Compilers/Core/Portable/MetadataReader/SymbolFactory.cs

```diff
@@ -40,7 +40,8 @@ internal abstract class SymbolFactory<ModuleSymbol, TypeSymbol>
         internal abstract TypeSymbol GetSystemTypeSymbol(ModuleSymbol moduleSymbol);
         internal abstract TypeSymbol GetEnumUnderlyingType(ModuleSymbol moduleSymbol, TypeSymbol type);
 
-        internal abstract bool IsVolatileModifierType(ModuleSymbol moduleSymbol, TypeSymbol type);
+        internal abstract bool IsAcceptedVolatileModifierType(ModuleSymbol moduleSymbol, TypeSymbol type);
+        internal abstract bool IsAcceptedIsConstModifierType(TypeSymbol type);
```

Comment:
`IsAccepted___Modifier` indicates that this perticular type is accepted as a modifier by the current implementation (language).
The previous naming suggested that the implementation checks if the type matches or not, not whether the langauge supports it or not.

---
In reply to: [118752054](https://github.com/dotnet/roslyn/pull/19658#discussion_r118752054) [](ancestors = 118752054)